### PR TITLE
Add Explorables

### PR DESCRIPTION
**Pull request by sp4ke**
**https://github.com/sp4ke/awesome-explorables**

This is a curated list of explorable explanations. Explorable Explanations as defined by Bret Victor are rich interactive documents that allow the reader to explore and question the topic while learning. This list is officially supported by the /r/explorables reddit.

I am the maintainer of the list.  I made sure to read the contribution guidelines many times so please let me know if there are mistakes, last time I tried to submit this list it was denied without further details. **

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- The description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
> last time I tried to submit this list it was denied without further details

It was closed then because you never replied: https://github.com/sindresorhus/awesome/pull/952

File: readme.md [hunk 1]
```diff
@@ -417,6 +417,7 @@
 - [Programming for Kids](https://github.com/HollyAdele/awesome-programming-for-kids)
 - [Educational Games](https://github.com/yrgo/awesome-eg) - Learn while playing.
 - [JavaScript Learning](https://github.com/micromata/awesome-javascript-learning)
+- [Awesome Explorables](https://github.com/sp4ke/awesome-explorables) - Explorable explanations.
```
Comment (sindresorhus) [inline readme.md]: The description here doesn't really explain what it is about.

And you need to drop the `Awesome` prefix from the name.

**Comment by sindresorhus** [comment]
> Includes a succinct description of the project/theme at the top of the readme. (Example)

Not followed

> The description starts with an uppercase character and ends with a period.

Not followed

> Entries have a description, unless the title is descriptive enough by itself. It rarely is though.

Not followed, there are many items that are not clear enough without a description.

**Comment by sp4ke** [comment]
> The description here doesn't really explain what it is about.

I tried to add a description but you should know it does not cover what it really means before reading its formal definition.  

The definition is explained here by the inventor of the concept [Explorable Explanations](http://worrydream.com/ExplorableExplanations/). 

- I added a description at the top of the project as required.

> Entries have a description, unless the title is descriptive enough by itself. It rarely is though.

I don't think this is possible for all items, many of them are broad or complex enough to come up with a unique description other than the "Category" of the entry. Which is why I do my best to categorize the content. 

If you still don't want to approve those changes fair enough you can close my PR.

Thanks for your time.

**Comment by legodude17** [comment]
Try to get descriptions on everything.
Maybe add a LICENSE file.

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/sp4ke/awesome-explorables
✖ Linting

  readme.md:25:4
  ✖    1:1    Missing Awesome badge after the main heading                                             remark-lint:awesome/badge
  ✖   24:1    ToC item "Tools & Resources" does not match corresponding heading "Tools And Resources"  remark-lint:awesome/toc
  ✖   25:4    Incorrect list-item indent: remove 1 space                                               remark-lint:list-item-indent
  ✖   30:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   31:3    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   32:3    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   33:3    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   34:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   35:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   36:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   37:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   38:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   39:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   40:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   41:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   47:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   48:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   49:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   50:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   51:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   52:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   53:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   54:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   55:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   56:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   57:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   58:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   62:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   63:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   64:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   64:50   List item link and description must be separated with a dash                             remark-lint:awesome/list-item
  ✖   65:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   70:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   71:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   72:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   77:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   85:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   86:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   87:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   88:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   89:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   90:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   91:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   92:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   93:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   94:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   95:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   96:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   97:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   98:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   99:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖   99:3    Link to https://classes.soe.ucsc.edu/cmps179/Spring13/#Week9 is dead                     remark-lint:no-dead-urls
  ✖  100:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  101:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  106:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  107:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  108:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  109:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  110:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  111:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  112:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  116:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  117:2    Don’t pad emphasis with inner spaces                                                     remark-lint:no-inline-padding
  ✖  118:2    Don’t pad emphasis with inner spaces                                                     remark-lint:no-inline-padding
  ✖  119:4    List item description must end with proper punctuation                                   remark-lint:awesome/list-item
  ✖  120:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  121:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  122:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  123:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  124:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  125:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  125:57   List item description must end with proper punctuation                                   remark-lint:awesome/list-item
  ✖  130:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  131:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  137:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  138:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  139:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  140:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  141:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  142:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  143:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  147:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  148:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  148:4    Text "github" should be written as "GitHub"                                              remark-lint:awesome/spell-check
  ✖  149:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  150:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  151:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  152:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  153:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  154:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  159:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  160:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  160:186  List item link and description must be separated with a dash                             remark-lint:awesome/list-item
  ✖  161:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  165:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  166:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  167:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  167:54   List item description must end with proper punctuation                                   remark-lint:awesome/list-item
  ✖  168:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  168:55   List item link and description must be separated with a dash                             remark-lint:awesome/list-item
  ✖  169:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  173:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  174:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  178:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  179:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  180:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  181:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  182:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  183:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  188:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  189:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  190:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  191:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  192:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  193:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  194:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  195:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  200:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  201:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  202:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  203:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  204:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  205:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  205:51   List item description must end with proper punctuation                                   remark-lint:awesome/list-item
  ✖  206:3    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  207:3    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  208:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  208:3    Link to https://www.geogebra.org/materials/ is dead                                      remark-lint:no-dead-urls
  ✖  209:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  210:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  211:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  212:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  213:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  213:3    Invalid list item link URL                                                               remark-lint:awesome/list-item
  ✖  214:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  215:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  216:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  217:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  217:42   List item link and description must be separated with a dash                             remark-lint:awesome/list-item
  ✖  218:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  219:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  219:73   List item link and description must be separated with a dash                             remark-lint:awesome/list-item
  ✖  220:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  220:3    Invalid list item link URL                                                               remark-lint:awesome/list-item
  ✖  221:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  221:43   List item link and description must be separated with a dash                             remark-lint:awesome/list-item
  ✖  226:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  226:49   List item description must end with proper punctuation                                   remark-lint:awesome/list-item
  ✖  227:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  227:75   List item description must end with proper punctuation                                   remark-lint:awesome/list-item
  ✖  228:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  229:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  229:64   List item description must end with proper punctuation                                   remark-lint:awesome/list-item
  ✖  230:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  231:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  232:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  233:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  234:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  235:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  236:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  244:1    Don’t add a trailing ! to headings                                                       remark-lint:no-heading-punctuation
  ✖  248:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style
  ✖  249:1    Marker style should be -                                                                 remark-lint:unordered-list-marker-style

  163 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
> I don't think this is possible for all items, many of them are broad or complex enough to come up with a unique description other than the "Category" of the entry. Which is why I do my best to categorize the content.

Sure, but many are possible. Wikipedia is your friend. For example for the `Primer on Bézier Curves` entry, the description could be `- Widely used in computer graphics to model smooth curves.`.

File: readme.md [hunk 2]
```diff
@@ -417,6 +417,7 @@
 - [Programming for Kids](https://github.com/HollyAdele/awesome-programming-for-kids)
 - [Educational Games](https://github.com/yrgo/awesome-eg) - Learn while playing.
 - [JavaScript Learning](https://github.com/micromata/awesome-javascript-learning)
+- [Explorable Explanations](https://github.com/sp4ke/awesome-explorables) - Learn through active reading and reactive documents as defined by Bret Victor.
```
Comment (sindresorhus) [inline readme.md]: I would drop the `as defined by Bret Victor.` part as not everyone would know how that is, and if they know, then explorables would be obvious to them anyway.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Comment by sp4ke** [comment]
Sorry I missed the notifs. I will update this issue later when I do the updates.

**Pull request closed without merging**